project/src/main/scala/org/broadinstitute/clio/publish/ClioPublishingPlugin.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.clio.publish
+
+import sbt._
+import sbt.Keys._
+
+/**
+  * A plugin for adding the settings required for clio subprojects to Artifactory.
+  *
+  * Enable this plugin on every sub-project that should be published.
+  */
+object ClioPublishingPlugin extends AutoPlugin {
```

Comment:
I don't think adding `Clio` to this class adds to its description, as we don't have any other publishing plugins, and the class is already in a package/project called clio.

A more descriptive name might be `ArtifactoryPublishingPlugin`.